phpstan.neon

```diff
@@ -3,8 +3,9 @@ parameters:
 	paths:
 		- src
 		- tests
-	bootstrapFiles:
-		- vendor/bin/.phpunit/phpunit/vendor/autoload.php
+	# @todo use below if/when we are back on symfony/phpunit-bridge
```

Comment:
With the strategy previously mentioned, we can revert the changes in this file.

Reply:
done: https://github.com/arderyp/panther/commit/3c0c3f6f7d300bacdd4c1ce5fd42a10018edf07f

@dunglas please resolve if sufficient